### PR TITLE
Fix update.sh to update ambrosus nodes properly

### PR DESCRIPTION
**Pull request by vpowler**

### CODE REVIEW

File: update.sh [hunk 1]
```diff
@@ -13,6 +13,34 @@ END
 sysctl -p /etc/sysctl.d/10-ambrosus.conf
 
 git pull origin master
+
+###
+
+ # install or update nvm
+wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
```
Comment (valar999) [inline update.sh:20]: This will not upgrade, only install :)

File: update.sh [hunk 2]
```diff
@@ -13,6 +13,34 @@ END
 sysctl -p /etc/sysctl.d/10-ambrosus.conf
 
 git pull origin master
+
+###
+
+ # install or update nvm
```
Comment (valar999) [inline update.sh:19]: Why this is here "###" and a space before "# install"?

File: update.sh [hunk 3]
```diff
@@ -13,6 +13,34 @@ END
...
+wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
+source $HOME/.nvm/nvm.sh && \. $HOME/.nvm/nvm.sh
```
Comment (valar999) [inline update.sh:21]: "source" and "." are identical.
"." don't need ""

File: update.sh [hunk 4]
```diff
@@ -13,6 +13,34 @@ END
...
+echo "The $(command -v nvm) has been installed."
```
Comment (valar999) [inline update.sh:22]: Is there a reason to use "command"?

File: update.sh [hunk 5]
```diff
@@ -13,6 +13,34 @@ END
...
+DEFAULT_NODE_VERSION="14"
+
+ # check required node.js version
+REQUIRED_NODE_VERSION=$(cat $PWD/package.json | grep -Eo '"node": "[^0-9][^0-9]?[0-9]+?' | grep -Eo '[0-9]+')
+if [ "$REQUIRED_NODE_VERSION" = "" ]; then
+    REQUIRED_NODE_VERSION=$DEFAULT_NODE_VERSION
+fi
+
+ # check system wide node.js version
+SYSTEM_NODE_VERSION=$(node -v | grep -Eo 'v[0-9]+' | cut -b 2-3)
```
Comment (valar999) [inline update.sh:34]: It's better to split version with "." and get first part.
`cut -b 2-3` not going to work with `v9.0.0` and `v100.0.0` :)

File: update.sh (hunk 2 above)
Comment (vpowler) [reply update.sh:19]: It's easier to read. Should be removed after the merge.

**Pull request merged**